projects/igniteui-angular/package.json

```diff
@@ -79,6 +77,10 @@
         "igniteui-theming": "^4.4.0",
         "@igniteui/material-icons-extended": "^3.0.0"
     },
+    "optionalDependencies": {
+        "@types/hammerjs": "^2.0.40",
+        "hammerjs": "^2.0.8"
+    },
```

Comment:
Hm, oddly enough [`optionalDependencies`](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#optionaldependencies) seems to still be installed by default. Perhaps that's intended or it matters much, but perhaps we could even not list hammer at all in that case? 
//CC @kdinev

Reply:
OK, so from the documentation, I see that the correct way to specify the hammer dependency is as a `peerDependency` with `peerDependencyMeta` as optional: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#peerdependenciesmeta